### ISSUE TITLE
replace daisy UI with react-spectrum

### ISSUE THREAD

**Issue by ryanfavour4** [TaiwoIgbaro1212/xp-portal__revised#3]
**Run `yarn remove daisyui` and  remove every component and feature specific to daisy ui. ignore the tailwind css as it would be useful in minor cases. install react spectrum with `yarn add @adobe/react-spectrum` and add few components from spectrum to see its use in action.**

@Joshthefullstack 
@Thi-mee 
@ryanfavour4

**Comment by Thi-mee** [comment]
This Looks fun. I want in on this

**Comment by ryanfavour4** [comment]
> This Looks fun. I want in on this

YUP!... if possible when you are free we could go over the folder. its quite straight foward once i explain.
